### DISCUSSION TITLE
Welcome to Debian-Pi-Aarch64 Discussions!

### DISCUSSION THREAD

**Discussion by aiminickwong** [openfans-community-offical/Debian-Pi-Aarch64#161] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TorstenH82** [comment]
Upgraded to 5.10.42 a few minutes before. Thank you very much for providing the upgrade package on mega.
